onnxruntime/core/providers/iree/iree_execution_provider.cc

```diff
@@ -133,20 +145,27 @@ common::Status IREEExecutionProvider::Compile(const std::vector<FusedNodeAndGrap
   if (auto* err = ireeCompilerOutputOpenMembuffer(&vmfb_output.output)) {
     return iree_ep_jit::ErrorToStatus(err, "Failure opening compiler output buffer: ");
   }
-  ORT_RETURN_IF_ERROR(inv.CompileAndOutputVMFB(vmfb_output.output));
+
+  // This will save the compiled module to current working directory.
+  if (info_.find("save_to") == info_.end())
+    info_["save_to"] = "";
```

Comment:
It might be better to have this save to a temp directory by default, rather than saving to the current working directory.